src/scenes/PlantingSitesRouter/view/BoundariesAndZones.tsx

```diff
@@ -158,23 +158,25 @@ function PlantingSiteMapView({ plantingSite, data, search }: PlantingSiteMapView
     }
   }, [data, search]);
 
-  const plantingSiteMapData = useMemo(() => {
-    return MapService.getMapDataFromAggregation({ ...plantingSite, plantingZones: data }).site;
+  const mapDataFromAggregarion = useMemo(() => {
+    return MapService.getMapDataFromAggregation({ ...plantingSite, plantingZones: data });
```

Comment:
*nit - not sure if we want to correct this typo: `Aggregarion` -> `Aggregation`

Reply:
Thanks!